cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -59,6 +59,7 @@ var (
 	retryIntervalStart     = flag.Duration("retry-interval-start", time.Second, "Initial retry interval of failed provisioning or deletion. It doubles with each failure, up to retry-interval-max.")
 	retryIntervalMax       = flag.Duration("retry-interval-max", 5*time.Minute, "Maximum retry interval of failed provisioning or deletion.")
 	workerThreads          = flag.Uint("worker-threads", 100, "Number of provisioner worker threads, in other words nr. of simultaneous CSI calls.")
+	csiTimeout             = flag.Duration("timeout", 10*time.Second, "Timeout for waiting for creation or deletion of a volume")
```

Comment:
nit: `operationTimeout`, `provisioningTimeout`, `createDeleteTimeout`, `rpcTimeout`?

Everything in this package is "csi" to me it does not seem descriptive enough for what it actually is.
The name of the flag itself too `timeout` is not very specific, considering we also have `connection-timeout`

Reply:
`--timeout` is already established in external-attacher and prefer consistency. For variable name, operationTimeout sounds good.